crt_portal/crt_portal/storage.py

```diff
@@ -0,0 +1,16 @@
+from django.core.files.storage import get_storage_class
+from storages.backends.s3boto3 import S3Boto3Storage
+
+
+class CachedS3Boto3Storage(S3Boto3Storage):
```

Comment:
I think we want `S3ManifestStaticStorage` to keep the file hashing/cache breaking.
```suggestion
from storages.backends.s3boto3 import S3ManifestStaticStorage


class CachedS3Boto3Storage(S3ManifestStaticStorage):
```